webhooks/finished/server.js

```diff
@@ -357,6 +324,60 @@ webhookApp.post("/server/plaid_webhook", async (req, res, next) => {
   }
 });
 
+function handleTransactionsWebhook(code, requestBody) {
+  if (code === "INITIAL_UPDATE") {
+    console.log(
+      `First patch of transactions are done. There's ${requestBody.new_transactions} available`
+    );
+  } else if (code === "HISTORICAL_UPDATE") {
+    console.log(
+      `Historical transactions are done. There's ${requestBody.new_transactions} available`
+    );
+  } else if (code === "DEFAULT_UPDATE") {
+    console.log(
+      `New data is here! There's ${requestBody.new_transactions} available`
+    );
+  } else if (code === "TRANSACTIONS_REMOVED") {
+    console.log(
+      `Looks like a few transactions have been reversed and should be removed from our records`
+    );
+  }
+}
+
+function handleAssetsWebhook(code, requestBody) {
+  if (code === "PRODUCT_READY") {
+    console.log(
+      `Looks like asset report ${requestBody.asset_report_id} is ready to download`
+    );
+  } else if (code === "ERROR") {
+    console.log(
+      `I had an error generating this report: ${requestBody.error.error_message}`
+    );
+  }
+}
+
+function handleItemWebhook(code, requestBody) {
+  if (code === "ERROR") {
+    console.log(
+      `I received this error: ${requestBody.error.error_message}| should probably ask this user to connect to their bank`
+    );
+  } else if (code === "NEW_ACCOUNTS_AVAILABLE") {
+    console.log(
+      `There are new accounts available at this Financial Institution! (Id:   ${requestBody.item_id}) We might want to ask the user to share them with us`
+    );
+  } else if (code === "PENDING_EXPIRATION") {
+    console.log(
+      `We should tell our user to reconnect their bank with Plaid so there's no disruption to their service`
+    );
+  } else if (code === "USER_PERMISSION_REVOKED") {
+    console.log(
+      `The user revoked access to this item. We should remove it from our records`
+    );
+  } else if (code === "WEBHOOK_UPDATE_ACKNOWLEDGED") {
+    console.log(`Hooray! You found the right spot!`);
+  }
+}
+
 const verifyWebhook = async (req) => {
```

Comment:
why not a switch statement for all these above if statements as well?

Reply:
Because I was lazy! (I'll fix it.)